addons/web/static/src/core/autocomplete/autocomplete.js

```diff
@@ -124,6 +124,10 @@ export class AutoComplete extends Component {
             return;
         }
 
+        if (this.props.resetOnSelect) {
```

Comment:
I don't understand this one, maybe too tired on Friday afternoon ^^

Can you explain with an example and some screenshots maybe?

Reply:
Sorry to insist, but I see there is a case where the option is "option.unselectable" right above your code, and seems to do a bit the same as your option (except it returns directly, but not sure it would matter since you don't select the option anyway?).

Shouldn't the use case "the tag is already created" go into that part?

I fear that we are adding a global option to solve a very specific use case.

Reply:
Do we need an option ? Feeling like this is more a fix and not really something actionable by people ?